File: Ejercicio 5/Ejercicio 5.cpp
#include <iostream>
using namespace std;

int main() {

    string NAME = "Renzo Leonardo Cienfuegos Cardenas";
    int GROUP = 2;

    int n;
    double numero;

    // Solicitamos la cantidad de elementos del vector
    cout << "Ingrese la cantidad de elementos del vector: ";
    cin >> n;

    double vector[n];
    double resultado[n];

    // Solicitamos el número real
    cout << "Ingrese el numero real que desea sumar al vector: ";
    cin >> numero;

    // Solicitamos los elementos del vector
    for (int i = 0; i < n; i++) {

        cout << "Ingrese el elemento " << i + 1 << ": ";
        cin >> vector[i];

        // Sumamos el número a cada componente del vector
        resultado[i] = vector[i] + numero;

    }

    // Mostramos el vector resultante
    cout << "\nVector resultante:\n";

    for (int i = 0; i < n; i++) {

        cout << resultado[i] << " ";

    }

    cout << endl;

    return 0;
}

File: Ejercicio 5/Ejercicio 5.py
NAME = "Renzo Leonardo Cienfuegos Cardenas"
GROUP = 2

# Solicitamos la cantidad de elementos del vector
n = int(input("Ingrese la cantidad de elementos del vector: "))

# Creamos los vectores
vector = []
resultado = []

# Solicitamos el número real
numero = float(input("Ingrese el número real que desea sumar al vector: "))

# Solicitamos los elementos del vector
for i in range(n):

    elemento = float(input(f"Ingrese el elemento {i + 1}: "))

    # Agregamos el elemento al vector
    vector.append(elemento)

    # Sumamos el número a cada componente del vector
    resultado.append(elemento + numero)

# Mostramos el vector resultante
print("\nVector resultante:")

for elemento in resultado:
    print(elemento, end=" ")
